framework/configuration/new_customer.rst

```diff
@@ -0,0 +1,35 @@
+.. _add_customer_module:
+
+Add new Customer module
+=======================
+
+ 1. Go into your nice git repository
+ 2. Create the new customer module.
+
+  .. code-block:: bash
+
+    mkdir customer/${CUSTOMER}
+    cd customer/${CUSTOMER}
+    ../../src/bin/mkappmodule.sh
+
+ 3. Generate the pom dependencies. In backoffice execute "Modul"->"Aktionen"->"Modulabhängigkeiten"->"POM generieren"
+
+  .. figure:: new_customer/modules.png
+        :scale: 60%
+
+ 4. Check the generated dependencies, there are probably some mistakes in there. Add the dependencies to the customers pom
+
+S3 module
+^^^^^^^^^
+
+Add this entry to the customer pom **if the customer should use S3**:
```

Comment:
> if the customer should use S3

Shouldn't S3 be enabled for all new customers? If not, can you clarify when it has to be enabled.

Reply:
Hey, you ignored my comment :(.

Reply:
What customers shouldn't use it?

Reply:
At the moment no customer should use it. But in a few weeks all of them should.